circle.yml

```diff
@@ -11,6 +11,7 @@ test:
     - cd $IMPORTPATH && go test -v -race $(go list ./... | grep -v /vendor/)
     - cd $IMPORTPATH && go vet $(go list ./... | grep -v /vendor/)
     - cd $IMPORTPATH && scripts/end2end_test.sh
+    - cd $IMPORTPATH && go test -run='^$' -bench=. $(go list ./... | grep -v /vendor/)
```

Comment:
don't those benchmarks fail anyway? do you mean the cassandra-idx and elasticsearch-idx working? seems like those require extra work (e.g. spinning up those services) to get working in the first place so i never really ran them.